configure

```diff
@@ -890,6 +897,19 @@ if (!$ace_src || !$tao_src) {
 print "Using ace_src: $ace_src\n" if $opts{'verbose'};
 print "Using tao_src: $tao_src\n" if $opts{'verbose'};
 
+if ($is_windows && $opts{'target'} eq 'android') {
+  # Rename the VERSION files in ACE, TAO, and OPENDDS. On Windows the cross
+  # compiler will pick them up instead of the version standard header the
+  # standard library is trying to include.
+  if (exists($opts{'dry-run'})) {
+    print("Would rename VERSION files to get around version header Windows issue.\n");
```

Comment:
```suggestion
    print "Would rename VERSION files to get around version header Windows issue.\n";
```

Reply:
I got to say I really don't like this aspect of Perl.

Reply:
I got to say I really don't like this aspect of Perl.